lib/vsc/mympirun/mpi/openmpi.py

```diff
@@ -106,13 +107,15 @@ def set_mpiexec_options(self):
             over = 'NO'
 
         # make sure we start enough per node so it can fill the total_number_of_processes
-        tnop = self.total_number_of_processes()
-        un = len(self.nodes_uniq)
-        ppn = int(math.ceil(tnop / un))
-        logging.debug("Setting up map for %s (%s total number of processes on %s unique nodes)", ppn, tnop, un)
+        tot_processes = self.total_number_of_processes()
+        unique_nodes = len(self.nodes_uniq)
+        processes_per_node = int(math.ceil(tot_processes / unique_nodes))
+        logging.debug("Setting up map for %s (%s total number of processes on %s unique nodes)",
+                      processes_per_node, tot_processes, unique_nodes)
 
+        # See https://www.open-mpi.org/doc/current/man1/mpirun.1.php "Mapping, Ranking, and Binding: Oh My!"
```

Comment:
@stdweird Let's change this to a version-specific URL, because this stuff will probably change in OpenMPI v5:

```suggestion
        # See https://www.open-mpi.org/doc/v4.1/man1/mpirun.1.php "Mapping, Ranking, and Binding: Oh My!"